roles/firmware/tasks/main.yml

```
@@ -0,0 +1,20 @@
+- name: Include older RPi Firmware
+  include_tasks: install.yml
+  when: RPi_firmware_retrieved is undefined
+
+- name: Install check-firmware service files
+  template:
+    src: "{{ item.src }}"
+    dest: "{{ item.dest }}"
+    mode: "{{ item.mode }}"
+  with_items:
+    - { src: 'fw-warn.sh', dest: '/usr/sbin/' mode: '0644' }
```

fw-warn.sh is a better name than fw_warn.sh below, if this is needed.
```suggestion
    - { src: 'fw-warn.sh', dest: '/usr/sbin/', mode: '0644' }
```